samtools.wdl

```diff
@@ -363,3 +366,32 @@ task View {
                       category: "advanced"}
     }
 }
+
+task FilterShortReadsBam {
+    input {
+        File bamFile
+        String outputPathBam
+        String dockerImage = "quay.io/biocontainers/samtools:1.8--h46bd0b3_5"
+    }
+
+    command {
+        set -e
+        mkdir -p "$(dirname ~{outputPathBam})"
+        samtools view -h ~{bamFile} | \
+        awk 'length($10) > 30 || $1 ~/^@/' | \
+        samtools view -bS -> ~{outputPathBam} 
+        samtools index ~{outputPathBam}
```

Comment:
The convention within miniwdl tends to be that BAM indexes only get the `.bai`, not `.bam.bai`. You should give an output path here as well.